### ISSUE TITLE
Placing Faeries in enclosure corrupts world

### ISSUE THREAD

**Issue by davqvist** [BenjaminSutter/genera#10]
Genera 0.0.7
Forge 2485
MC 1.12.1

When you place a faery in an enclosure, the game crashes and the world is corrupted.

https://pastebin.com/4xvu2BwM

**Comment by BenjaminSutter** [comment]
Odd. I fixed this and it worked fine for me last night. Let me try again. It's an easy fix; I just thought I'd already done it.

**Comment by BenjaminSutter** [comment]
This is fixed in the latest commit. I'll get a release out later tonight.

**Comment by BenjaminSutter** [comment]
Fixed in 0.0.8, just released.
